.drone.star

```diff
@@ -14,65 +14,82 @@ windowsglobalimage="cppalliance/dronevs2019"
 
 def main(ctx):
   return [
+  # Priorities:
+  # 1. Coverage
+  # 2. Latest compilers
+  # 3. Oldest compilers
+  # 4. Sanitizers
+  # 5. Documentation
+  # 6. Other platforms
+  # 7. The rest
```

Comment:
No!!! This is not what we wrote. Again:
```
  # Priorities: (no 2a, no betas)
  #
  # coverage
  # latest gcc: 11,14,17,20
  # latest clang: 11,14,17,20
  # oldest gcc: 4.8
  # oldest clang: 3.8
  # valgrind
  # asan
  # tsan
  # ubsan
  # docs
  # arm64
  # s390x
  # (...the rest)
```

Reply:
Oh... you really want the ugly version... ok